src/pages/admin/customerInstitutionFields/FeideOrganizationIdField.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Field, FieldProps, ErrorMessage } from 'formik';
+import { CustomerInstitutionFieldNames } from '../../../types/customerInstitution.types';
+import { TextField } from '@material-ui/core';
+
+const FeideOrganizationIdField: FC = () => {
+  const { t } = useTranslation('admin');
+
+  return (
+    <Field name={CustomerInstitutionFieldNames.FEIDE_ORGANIZATION_ID}>
+      {({ field, meta: { touched, error } }: FieldProps) => (
+        <TextField
+          {...field}
+          label={t('feide_organization_id')}
+          fullWidth
+          variant="outlined"
+          inputProps={{ 'data-testid': 'customer-institution-feide-organization-id-input' }}
+          error={touched && !!error}
+          helperText={<ErrorMessage name={field.name} />}
+        />
+      )}
+    </Field>
+  );
+};
+
+export default FeideOrganizationIdField;
```

Comment:
Er det nødvendig å opprette en egen komponent for disse tre Fields med TextField? Ser ut som det ikke er så mange forskjellige props? Kanskje lage en felleskomponent eller?

Reply:
Gjelder forsåvidt ShortNameField også

Reply:
Ble litt duplisering her ja... slår sammen `DisplayName`, `ShortName`, `ArchiveName` og `FeideId` til `CustomerInstitutionTextField`!